File: blockchain-web/web/frontend/src/helpers/qrcode-helper.js
export function generateOrderQRImg(qrCode) {
  return `https://api.qrserver.com/v1/create-qr-code/?size=100x100&data=OR!${qrCode}`;
}
export function generateWithdrawQRImg(qrCode) {
  return `https://api.qrserver.com/v1/create-qr-code/?size=100x100&data=WD!${qrCode}`;
}

File: blockchain-mobile/lib/4_helper/qrcode_helper.dart
class QrCodeHelper {
  static String getQrImage(String data, {required EQrType type}) {
    // return Uri.encodeFull(
    //     "https://api.qrserver.com/v1/create-qr-code/?size=100x100&data=${type.symbol}!$data");
    return Uri.encodeFull(
        "https://api.qrserver.com/v1/create-qr-code/?data=${type.symbol}!$data&size=100x100");
  }
}

enum EQrType { Order, Withdraw }

extension EQrTypeExt on EQrType {
  String get symbol => switch (this) {
        EQrType.Order => 'OR',
        EQrType.Withdraw => 'WD',
      };
}
